### PR TITLE
Better UX, project and build.sbt generation

### PR DESCRIPTION
**Pull request by boseabhishek**
- user can input project, scala-version and scalatest-version w/ defaults
- main and test files generated under chosen project structure (or default)

### CODE REVIEW

File: src/main/g8/default.properties [hunk 1]
```diff
@@ -1,2 +1,5 @@
 name=My Something Project
 description=Say something about this template.
+package=com.example
+scala-version=2.13.6
```
Comment (SethTisue) [inline src/main/g8/default.properties:4]: this can be bumped to 2.13.12

File: src/main/g8/default.properties [hunk 2]
```diff
@@ -1,2 +1,5 @@
...
+scalatest-version=3.2.9
```
Comment (SethTisue) [inline src/main/g8/default.properties:5]: and this can be bumped to whatever latest 3.2.x is

Comment (boseabhishek) [reply src/main/g8/default.properties]: Done

**Comment by SethTisue** [comment]
Thank you!

**Comment by boseabhishek** [comment]
> Thank you!

Thanks for accepting the PR :)

**Pull request merged**